tests/codegen/array-immediate-param-noundef.rs

```diff
@@ -0,0 +1,95 @@
+// Check that small array immediates that fits in target pointer size in argument position have
+// `noundef` parameter metadata. Note that the `noundef` parameter metadata is only applied if:
+// - `!arg.layout.is_unsized() && size <= Pointer(AddressSpace::DATA).size(cx)`
+// - optimizations are turned on.
+//
+// ignore-tidy-linelength
+//@ only-64bit (presence of noundef depends on pointer width)
+//@ compile-flags: -C no-prepopulate-passes -O
+#![crate_type = "lib"]
+
+// CHECK: define noundef i64 @replace_short_array_u64x1(ptr noalias noundef align 8 dereferenceable(8) %r, i64 noundef %0)
```

Comment:
Maybe you can change this to:

```suggestion
// CHECK: define noundef i64 @replace_short_array_u64x1(ptr {{.*}}, i64 noundef %{{.*}})
```

Reply:
Thanks, changed the test cases to use wildcard for stuff we don't care about.